### PR TITLE
Add automatic deployment / Fix branch behaviour

### PR DESCRIPTION
**Pull request by DiljotSG**
This PR fixes the branch behaviour.
Regenerating goldens *should* work on core now.

Additionally, this repo now automatically redeploys upon a merge to master 🚀 .

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -329,7 +329,9 @@ async function regenGoldens (context, branchName) {
           }]
         }
       },
-      branch: branchName,
+      branch: {
+        only: branchName
+      },
```
Comment (dlockhart) [inline index.js:334]: Will this override our `branches` setting?
https://github.com/BrightspaceUI/core/blob/master/.travis.yml#L3

Comment (dlockhart) [reply index.js:334]: Or should this be:
```
branches: null,
branch: {
  only: branchName
}
```

Comment (DiljotSG) [reply index.js:334]:
```

Comment (DiljotSG) [reply index.js:334]: Maybe I should make it:
```
branches: {
  only: branchName
}
```?

I want to make the simplest change that is most likely to work for most of the Brightspace repos.

Comment (DiljotSG) [reply index.js:334]: I want to make the simplest change that is most likely to work for most of the Brightspace repos.

Comment (dlockhart) [reply index.js:334]: Right. Yeah basically no matter what filter they have in their `.travis.yml` file, you want to get it to ignore that.

**Pull request merged**